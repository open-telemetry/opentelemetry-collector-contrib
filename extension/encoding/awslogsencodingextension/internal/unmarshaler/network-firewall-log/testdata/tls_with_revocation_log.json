{
    "firewall_name": "test-firewall",
    "availability_zone": "us-east-1a",
    "event_timestamp": "2025-10-20T10:30:45.123Z",
    "event": {
        "event_type": "tls",
        "flow_id": 1234567892,
        "src_ip": "192.168.1.100",
        "src_port": 54323,
        "dest_ip": "10.0.1.52",
        "dest_port": 443,
        "proto": "TCP",
        "sni": "example.com",
        "revocation_check": {
            "leaf_cert_fpr": "1234567890EXAMPLE0987654321",
            "action": "DROP",
            "status": "REVOKED"
        },
        "tls_error": {
            "error_message": "Certificate verification failed"
        },
        "tls": {
            "subject": "CN=example.com",
            "issuer": "CN=Let's Encrypt Authority X3,O=Let's Encrypt,C=US",
            "session_resumed": true
        }
    }
}
