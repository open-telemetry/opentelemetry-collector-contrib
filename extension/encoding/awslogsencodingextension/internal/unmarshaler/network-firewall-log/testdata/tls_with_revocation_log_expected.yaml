resourceLogs:
  - resource:
      attributes:
        - key: cloud.provider
          value:
            stringValue: aws
        - key: aws.networkfirewall.name
          value:
            stringValue: test-firewall
        - key: cloud.availability_zone
          value:
            stringValue: us-east-1a
    scopeLogs:
      - scope:
          name: github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/awslogsencodingextension
          version: ""
          attributes:
            - key: encoding.format
              value:
                stringValue: aws.networkfirewall
        logRecords:
          - observedTimeUnixNano: "0"
            timeUnixNano: "1760956245123000000"
            attributes:
              - key: aws.networkfirewall.event.type
                value:
                  stringValue: tls
              - key: aws.networkfirewall.flow_id
                value:
                  intValue: "1234567892"
              - key: source.address
                value:
                  stringValue: 192.168.1.100
              - key: source.port
                value:
                  intValue: "54323"
              - key: destination.address
                value:
                  stringValue: 10.0.1.52
              - key: destination.port
                value:
                  intValue: "443"
              - key: network.transport
                value:
                  stringValue: TCP
              - key: server.address
                value:
                  stringValue: example.com
              - key: aws.networkfirewall.tls.revocation_check.leaf_cert_fpr
                value:
                  stringValue: 1234567890EXAMPLE0987654321
              - key: aws.networkfirewall.tls.revocation_check.action
                value:
                  stringValue: DROP
              - key: aws.networkfirewall.tls.revocation_check.status
                value:
                  stringValue: REVOKED
              - key: aws.networkfirewall.tls.error.message
                value:
                  stringValue: Certificate verification failed
              - key: tls.client.subject
                value:
                  stringValue: CN=example.com
              - key: tls.client.issuer
                value:
                  stringValue: CN=Let's Encrypt Authority X3,O=Let's Encrypt,C=US
              - key: tls.resumed
                value:
                  boolValue: true
